app/components/homepage/SectionHero.vue

```diff
@@ -0,0 +1,65 @@
+<template functional>
+  <header class="section-hero">
+    <h1 class="section-hero__title">En 2022,<br />Nous sommes de retour !</h1>
+    <p class="section-hero__place">
+      On se retrouve au Palais des Congrès <br />
+      de Bordeaux Lac
+    </p>
+    <time class="section-hero__date" datetime="2022-12-02">02/12/2022</time>
+    <h2 class="section-hero__invitation">Et on a hâte de vous y (re)voir !</h2>
+  </header>
+</template>
+
+<script>
+export default {
+  name: "SectionHero",
+};
+</script>
+
+<style lang="scss" scoped>
+.section-hero {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $primary-dark;
+  color: $light-font;
+  text-align: center;
+  padding: 100px 0;
```

Comment:
🍰 Maybe we could have a spacing system to ensure we have consistent spacings?

Reply:
Bonne idée !